src/strategies/GetEventStartTimeStrategy.ts

```diff
@@ -0,0 +1,25 @@
+import i18next from 'i18next';
+import { Context } from 'telegraf';
+import { ChatData } from 'Bot';
+import { DateType } from 'Event';
+import Strategy from 'strategies/Strategy';
+import GetEventEndDateStrategy from 'strategies/GetEventEndDateStrategy';
+
+export default class GetEventStartTimeStrategy extends Strategy {
+    constructor(ctx: Context) {
+        super();
+
+        ctx.reply(i18next.t('strategy.getStartTime'));
+    }
+
+    public execute(ctx: Context, chatData: ChatData) {
+        const eventDate = ctx.update.message?.text;
+
+        if (!eventDate) return;
+
+        chatData.event.setTime(eventDate, DateType.Start);
```

Comment:
in all executes I see the same pattern, what about somehow unify it? maybe move some logic to the Strategy Class? and maybe replace
```
chatData.event.setTime(eventDate, DateType.Start);
```
some unified way?